test/gurka/gurka.cc

```diff
@@ -310,11 +310,7 @@ inline void build_pbf(const nodelayout& node_locations,
     auto way_id = osm_id++;
     auto found = way.second.find("osm_id");
     if (found != way.second.cend()) {
-      uint64_t id = std::stoull(found->second);
-      if (id < osm_id) {
-        throw std::invalid_argument("Osm way id has already been used");
-      }
-      way_id = id;
+      way_id = std::stoull(found->second);
```

Comment:
So I needed a gurka test when multiple edges have the same way id. And this exception used to be thrown. Let me know if there's a better way to get such gurka test(in case you want to revert this piece of code)

Reply:
@merkispavel we have added exception tests for the path and eta like this [here](https://github.com/valhalla/valhalla/blob/master/test/gurka/gurka.cc#L1002)   But, it you don't think it will be useful for more tests in the future, I am fine how it is now.